src/ApiKitClient.ts

```diff
@@ -61,12 +61,12 @@ export class ApiKitClient {
     return this.instance!.post<T>(endpoint, data);
   }
 
-  public static async put<T>(endpoint: string, data: T): Promise<AxiosResponse<T>> {
+  public static async put<T>(endpoint: string, data: Partial<T>): Promise<AxiosResponse<T>> {
```

Comment:
PUT is usually not partial update, but we can keep it as laravel treats PUT & PATCH as an `update` request...